packages/elements/src/select/__test__/utils.js

```diff
@@ -11,8 +11,7 @@ export const snapshotIgnore = {
  */
 export const openedUpdated = async (el) => {
   await elementUpdated(el);
-  await nextFrame();
-  await nextFrame(); // IE11 needs a second iframe, otherwise resize observer is not run;
+  await nextFrame(2);
```

Comment:
Is it still need two frames?

Reply:
Yes. Otherwise, some test cases would fail on all 3 engines.